CONTRIBUTING.md

```diff
@@ -39,9 +37,31 @@ deep diving into the code base.
 Compiled binaries (for C libraries and Windows service wrapper) are committed to
 git to make build of development process Java-centric and simplified.
 
-# Local setup
+## Find suitable Sissues 
+
+Our maintainers label issues with relevant categories so you can use that to search for certain type of issues you'd like to work on. 
+If you don't know where to start, try search for issues labeled with `good first issue` or `help wanted`. 
+If you wish to understand how our maintainers work together, you can refer to [this section](#for-maintainers).
+
+
+# Environment setup
+
+## Requirements
+
+- Operating system - **x86-64**: Windows, Linux, FreeBSD and OSX
+- Java 11 64-bit
+- Maven 3 (from your package manager on Linux / OSX
+  ([Homebrew](https://github.com/Homebrew/brew)) or
+  [from the jar](https://maven.apache.org/install.html) for any OS)
+- Node.js 12 / npm 6 (to manage your Node.js versions we recommend
```

Comment:
nit: I think it's already Node.js 16 and npm 7. @insmac should know for sure.

Reply:
In my view our requirements should be:
`Node.js 16 / npm 8` (I would make `8` required as it's shipped with npm with the exception of FreeBSD which has these decoupled)